community/becoming-a-maintainer.md

```diff
@@ -1,3 +1,77 @@
 # Becoming a maintainer
 
-TODO
+## Introduction
+
+Our Maintainers are the people within Exercism that manage the different parts of our ecosystem. 
+We aim to have at least active two maintainers for each programming language track, and at least one active maintainer for all the parts of the platform.
+
+We have a formalised process for becoming and staying a maintainer, to make it achievable and fair for anyone.
+**You can be become a maintainer!**
+Read on to understand the responsibilities and the process.
+
+_Note: the sections below generally discuss Track Maintainers. 
+The process for becoming a maintainer of a different part of the ecosystem is similar but a little looser._
+
+## Responsibilities
+
+Maintainers are responsible for ensuring that a Track works well within Exercism, and continues to grow and develop. 
+That specifically means that maintainers:
+- **Set technical direction for a track:** Choosing the right language versions, tooling, etc.
+- **Manage the syllabus of a track:** Designing, choosing and defining the right Concepts and Exercises for a Track, and ensuring they are linked and structured appropriately.
+- **Ensure a traack track work well:** Ensuring the Track is up to date, that its test-runner is fast, etc.
+- **Ensure the track conforms to Exercism's standard:** Exercism has specifications and standards for each language Track to ensure a degree of consistency within the platform. While maintainers have free reign over much of the Track, all Tracks must conform to Exercism's standards.
+
+Maintainers are also responsibile for interacting with our contributor community.
+They review PRs from community, acting as gatekeepers, shephards, and cheerleaders for our community.
+They are responsible for keeping documentation and contribution guidelines up to date, and ensuring that there are relevant open issues for potential contributors to action.
+
+It is essential that Maintainers manifest Exercism's values of empathy and kindness, while maintaining high standards for the Track and the platform.
```

Comment:
```suggestion
It is essential that Maintainers manifest Exercism's values of empathy and kindness.
```

I'd leave out the second half or rephrase it. To me it sounds like these are somewhat contradictory the way it's currently phrased.

Reply:
I think often "being kind" can be associated with "letting anything go", so I want to have something which says the person's job is to enforce high standards in a kind and empathetic way.

Reply:
> I think often "being kind" can be associated with "letting anything go"

I agree. But I think the current phrasing reinforces that (imo false) association rather than acknowledging it and stating that it's not correct.

Perhaps something along the lines of "It is essential that Maintainers maintain high standards for the Track and the platform. This includes their commitment to Exercism's values, such as empathy and kindness."?